regparser/federalregister.py

```diff
@@ -25,7 +31,10 @@ def fetch_notice_json(cfr_title, cfr_part, only_final=False,
         params["conditions[type][]"] = 'RULE'
     if max_effective_date:
         params["conditions[effective_date][lte]"] = max_effective_date
-    response = requests.get(API_BASE + "articles", params=params).json()
+    url = API_BASE + "articles"
+    logger.info("Fetching notices - URL: %s Params: %r", url, params)
```

Comment:
We talked about this briefly, so documenting here:
I misunderstood when `requests` writes logs. We're all good